api/migrations/20201208212602-addPublicToMaster.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const { MASTER_SCHEMA_NAMES } = require('../src/utils/constants/misc');
+
```

Comment:
Given this is a migration to be run at a specific time and not in the future if those misc constants change, I think we should be more explicit here instead of referencing.

Reply:
Fixed in https://github.com/bcgov/NRPTI/pull/707/commits/57bddd711f91628562e2f6bb92e7153c7660f420